ci/cargo-config

```diff
@@ -0,0 +1,31 @@
+# Configuration of which linkers to call on Travis for various architectures
+
+[target.x86_64-unknown-linux-gnu]
+linker = "gcc"
+
+[target.x86_64-unknown-linux-musl]
+linker = "gcc"
+
+[target.i686-unknown-linux-gnu]
+linker = "gcc"
+
+[target.i686-unknown-linux-musl]
+linker = "gcc"
```

Comment:
is this the default? if so we should maybe leave it out in case Rust ever changes to lld?